sdk-archive.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: Flutter SDK Archive
+
+permalink: /sdk-archive/
+---
+
+The Beta channel contains the most stable Flutter builds. See [Flutter’s
+channels](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels)
+for details.
+
+<div id="tab-set-os" class="sdk-archives">
+	<ul class="tabs__top-bar">
+		<li class="tab-link current" data-tab="tab-os-windows">Windows</li>
+		<li class="tab-link" data-tab="tab-os-macos">Mac OS</li>
```

Comment:
Not that Apple spells it "macOS" on all their materials.

Reply:
Fixed!